src/main/java/ru/javarush/quest/entity/Account.java

```diff
@@ -0,0 +1,64 @@
+package ru.javarush.quest.entity;
+
+public class Account {
+    private Long id;
+    private final String name;
+    private final String login;
+    private String password;
+
+    public Account(Long id, String name, String login, String password) {
+        this.id = id;
+        this.name = name;
+        this.login = login;
+        this.password = password;
+    }
+
+    public Long getId() {
+        return id;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public String getLogin() {
+        return login;
+    }
+
+    public void setId(Long id) {
+        this.id = id;
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
```

Comment:
https://google.github.io/styleguide/javaguide.html#s4.1.1-braces-always-used
Braces are used with if, else, for, do and while statements, even when the body is empty or contains only a single statement.

Reply:
i deleted this equals method.